packages/itinerary-body/src/styled.tsx

```diff
@@ -362,17 +366,21 @@ export const LineBadgeContainer = styled.div`
 
 export const LineColumn = styled.div`
   /* flexbox column */
-  flex: 0 0 50px;
+  grid-column-start: 2;
+  grid-row: span 2;
   padding-right: 5px;
 `;
 
-export const LegDetails = styled.span``;
+export const LegDetails = styled.span`
+  display: grid;
+  grid-template-columns: 100px auto;
+`;
 
-export const PlaceRowWrapper = styled.div`
+export const PlaceRowWrapper = styled.section`
```

Comment:
This sounds more like an entry within an ordered list of steps, what do you think? It is like writing the steps of a recipe. `ItineraryBody` would be that ordered list.

Reply:
I'd agree with that! So removing `section` in favor of an `li`, and transforming the ItineraryBody into an `ol`?

Reply:
You got it!